### ISSUE TITLE
Handle TrueBlocks core type errors

### ISSUE THREAD

**Issue by tjayrush** [TrueBlocks/trueblocks-explorer#50]
TrueBlocks core returns a valid response even in the case of an error. This is the result of the way the command line versions of the code works and will not change in the future.

For example, this curl request:

```
curl "http://localhost:8080/transactions?transaction=12"
```

Does not complain even though the transaction does not exist. In this case, it returns valid JSON thus:

````
{
  "errors": [
    "chifra transactions - 12 does not appear to be a valid transaction index."
  ]
}
```

We should handle this case as an error and report the message returned in `@modules/core` I think.

**Comment by dszlachta** [comment]
This is because the response code is always 200. Just changing it to a correct error code should trigger `modules/core` error handling mechanism (but you still need to call `createErrorNotification` in the consumer-component to show anything). Here is the list of possible status codes: https://httpstatuses.com/. Lazy solution would be to check for `errors` in Go code and change status code to 400 each time the string is present. But ideally, in this particular case, it would be 404 - not found

**Comment by tjayrush** [comment]
Thanks @dszlachta Actually, the example above does return a 404 error -- that's handled by the Go server code because there is no route for transactions without an 's'. There are cases where we return 'error' data but a 200 return code though. I can fix that in the backend.

**Comment by tjayrush** [comment]
This is now implemented, but note #91
